README.md

```diff
@@ -35,6 +35,53 @@ Foundry consists of:
 
 ![Demo](.github/assets/demo.gif)
 
+---
+
+## Differences Between Foundry and Foundry-Polkadot Fork
+
+This section outlines the key differences between the original Foundry toolkit ([Foundry](https://github.com/foundry-rs/foundry)) and the ParityTech Foundry-Polkadot fork ([Foundry-Polkadot](https://github.com/paritytech/foundry-polkadot)). **It is intended for developers already familiar with Foundry who are interested in using the fork for Polkadot blockchain development.** The focus is on new features, changes, and unsupported functionalities in the fork, emphasizing its adaptation for Polkadot.
+
+### 1. Resolc Compiler Integration
+
+**Purpose**: The most significant addition in the fork is support for compiling Solidity contracts for Polkadot using the Resolc compiler, which targets PolkaVM in the `pallet-revive` framework ([Revive Repository](https://github.com/paritytech/revive)). This contrasts with the original Foundry's use of the Solc compiler for Ethereum's EVM.
```

Comment:
The main addition in the foundry-polkadot is support for compiling Solidity contracts targeting Polkadot, using the Resolc compiler to generate PolkaVM-compatible bytecode. This contrasts with the original Foundry's use of the Solc compiler for Ethereum's EVM.